plugins/pipelines/metrics.js

```diff
@@ -73,11 +91,13 @@ module.exports = () => ({
                 id: pipelineIdSchema
             }),
             query: schema.api.pagination.concat(
-                joi.object({
-                    startTime: joi.string().isoDate(),
-                    endTime: joi.string().isoDate(),
-                    aggregateInterval: joi.string().valid('none', 'day', 'week', 'month', 'year')
-                })
+                joi
+                    .object({
+                        startTime: joi.string().isoDate(),
+                        endTime: joi.string().isoDate(),
+                        aggregateInterval: joi.string().valid('none', 'day', 'week', 'month', 'year')
+                    })
+                    .unknown(true)
```

Comment:
can status & jobs keys, be added here with optional presence ?

Reply:
added

Reply:
is `unkown` still needed ?

Reply:
removed